apps-rendering/src/components/editions/starRating/index.tsx

```diff
@@ -54,9 +55,11 @@ const containerStyles = css`
 `;
 
 const StarRating: FC<Props> = ({ item }) =>
-	item.design === ArticleDesign.Review ? (
-		<div css={containerStyles}>{stars(item.starRating)}</div>
-	) : null;
+	item.design === ArticleDesign.Review
+		? maybeRender(item.starRating, (starRating) => (
+				<div css={containerStyles}>{stars(starRating)}</div>
+		  ))
+		: null;
```

Comment:
This ternary conditional may be redundant because `maybeRender` already deals with the possibility of `starRating` not being there, but it's there to enforce the rendering of stars only if `Design === Review` (is it even possible to have stars on non-review articles?)

Reply:
> (is it even possible to have stars on non-review articles?)
I hope not.

Reply:
Ah, if you don't perform this check does TypeScript complain that `starRating` may not exist on `Item`?

Reply:
Good shout - it does complain! So it's not a bad idea to keep the ternary there.